ci/scripts/github/github_commenter.py

```diff
@@ -18,8 +18,14 @@
 
 import re
 import logging
+import sys
+from pathlib import Path
 from typing import Dict, Tuple, Any, Optional, List, Union
 
+# Hackery to enable importing of utils from ci/scripts/jenkins
```

Comment:
i think at some point we were going to make this a Python package, right? is the idea that this hack bridges the gap til that happens?

Reply:
Yes, I think that's needed until we create the proper package structure.